vault.sh

```diff
@@ -302,31 +335,17 @@ vault_data() {
     cat $HOME/.config/libsh/hc_vaults.json
 }
 
+vault_config() {
+    echo $HOME/.config/libsh/hc_vaults.json
+    cat $HOME/.config/libsh/hc_vaults.json
+}
+
 vault_list() {
     local help=$(cat <<HELP
 ## vault_list
 
 List known vaults from ~/.config/libsh/hc_vaults.json
 
-example:
```

Comment:
Whats the reason for removing the example?

Reply:
Mistake.